subsys/net/ip/net_pkt.c

```diff
@@ -360,22 +354,20 @@ struct net_pkt *net_pkt_get_reserve(struct k_mem_slab *slab,
 	net_pkt_alloc_add(pkt, true, caller, line);
 
 #if NET_LOG_LEVEL == LOG_LEVEL_DBG
```

Comment:
Noticed that this should have been `CONFIG_NET_PKT_LOG_LEVEL >= LOG_LEVEL_DBG`, can you fix it here